db/migrate/20150330221903_change_requirements_for_reviews_table.rb

```diff
@@ -0,0 +1,19 @@
+class ChangeRequirementsForReviewsTable < ActiveRecord::Migration
+  def up
+    change_table :reviews do |t|
+      t.change :body, :string, {:null => false}
+      t.change :rating, :integer, {:null => false}
```

Comment:
Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.